website/docs/apps/controller_app/README.md

```diff
@@ -0,0 +1,172 @@
+---
+id: controller-app
```

Comment:
```suggestion
id: autoplot-app
```
+ where applicable & rename file `autoplot_app/README.md`

Reply:
Let's do a follow-up PR to scrub controller references.